compiler/src/main/resources/compiling-programs/for-loop-1.cell

```diff
@@ -0,0 +1,19 @@
+world {
+    size = 100 [wrap], 200 [wrap]
+    tickrate = 1
+    cellsize = 5
+}
+
+state ident (0, 0, 0) {
+    let x = 0;
+    let even = 0;
+    
+    for (x = 5; x >= 0; x = x -1) {
```

Comment:
Missing example of for statement with variable declaration and array lookup

Reply:
Variable declaration in init: 45b2453. Does not compile though and have added #103 on this.

Reply:
What do you mean with 'array lookup'? As the condition? Initialisation?

Reply:
In the initialization part, and by "array lookup" i mean `arr[2]`

Reply:
I think I understand what you mean, please look at examples 2 and 3 in 186acd9.

Reply:
Note workaround in eff57f5

Reply:
Something is not quite right about this https://github.com/d409f19/cellmata/commit/186acd96e669d8205ee3313a636d1bd0c02dc3db#r33360961